app/models/posts.rb

```diff
@@ -0,0 +1,12 @@
+class Posts < ApplicationRecord
+  belongs_to :users
+  has_many :likes, :comments
+
+  def update_post_counter
+    users.posts_counter += 1
+  end
```

Comment:
## Optional
- [ ] I suggest that you use [Rails active callbacks](https://guides.rubyonrails.org/active_record_callbacks.html), `after_save` to call the #update_post_counter method every time a new post has been saved to database, then make the `update_post_counter` method private

Reply:
@DeliceLydia thank you very much I appreciate it. I'm going to read about it and implement it